index.html

```diff
@@ -0,0 +1,180 @@
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(3).png" alt=""></div>
+          <div class="primary-text">
+            <h2>Multi-Post <br>Stories</h2>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-myself">
+      <div class="primary-text">
+        <h2>About <br>Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+           of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social">
+        <div class="let-connect"><h3>LET'S CONNECT</h3></div>
+        <ul class="social-icons">
+          <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
+          <li><a href=""><img src="./images/linked.svg" alt=""></a></li>
+          <li><a href=""><img src="./images/medium.svg" alt=""></a></li>
+          <li><a href=""><img src="./images/github.svg" alt=""></a></li>
+          <li><a href=""><img src="./images/angellist.svg" alt=""></a></li>
+        </ul>
+      </div>
+      <div class="normal-button">
+        <button type="button">Get my resume</button>
+      </div>
+      <div>
+        <ul class="language-list">
+          <li><img src="./images/Frame7.png" alt=""></li>
+          <li><img src="./images/Frame9.png" alt=""></li>
+          <li><img src="./images/Frame10.png" alt=""></li>
+          <li><img src="./images/Frame11.png" alt=""></li>
+          <li><img src="./images/Frame8.png" alt=""></li>
+          <li><img src="./images/Frame6.png" alt=""></li>
+        </ul>
+      </div>
+    </section>
+  </div>
+
+  <div class="button-wrapper">
+    <section class="button-section">
+      <ul class="button-actions">
+        <li><img src="./images/GuideHorizontal2.png" alt=""></li>
+        <li><img src="./images/GuideHorizontal4.png" alt=""></li>
+        <li><img src="./images/GuideHorizontal5.png" alt=""></li>
+        <li><img src="./images/GuideHorizontal6.png" alt=""></li>
+      </ul>
+      <div class="action-container">
+        <button type="button" class="action-buttons" id="enabled">Buttons</button>
+        <button type="button" class="action-buttons" id="hover">Button</button>
+        <button type="button" class="action-buttons" id="pressed">Button</button>
+        <button type="button" class="action-buttons" disabled id="disabled">Button</button>
+      </div>
+    </section>
+  </div>
```

Comment:
- [ ] You totally misunderstand the requirement, Here the previous reviewer asked to do style on the existing button like `Get my resume` & `See project` buttons, Kindly remove this part and add style on this button. Refer to [this](https://www.w3schools.com/howto/howto_css_animate_buttons.asp) to understand hover, active and disabled. Just learn the hover effect, for now, Plz don't fall into the rabbit hole.

Reply:
Thanks, @Deepakdanger, this is so specific and direct. I will do so right now!